### ISSUE TITLE
Creo que mi base de datos guardó tuplas del modelo anterior

### ISSUE THREAD

**Issue by FranLucchini** [IIC2513-2015-1/syllabus#14]
Ahora no me deja ni hacer rake a la semilla ni borrar la tupla... Alguna idea para ayudarme?

El error que me aparece es este:

ActiveRecord::StatementInvalid in EventsController#destroy
PG::UndefinedColumn: ERROR: column likes.event_id does not exist LINE 1: SELECT "likes".\* FROM "likes" WHERE "likes"."event_id" = $1 ^ : SELECT "likes".\* FROM "likes" WHERE "likes"."event_id" = $1

**Comment by lopezjurip** [comment]
Me imagino que la relación con `likes` es polimórfica. En tal caso, asegúrate de declararla como polimorphic y de respetar el esquema en la base de datos.

**Comment by FranLucchini** [comment]
Creo que ya entiendo a lo que te refieres. Ahora mi duda ha mutado...
Según yo agregar los comandos

``` ruby
has_one
has_many
belongs_to
```

a mis recursos es hacerlos polimórficos. Pero, al iniciar el server, me dice que las relaciones entre recursos no existen... Tengo que crearlas a mano?

Solo por si es importante: estamos usando Ruby on Rails 4

**Comment by FranLucchini** [comment]
P.D: lo del rake ya lo arreglé, el problema era un modelo anterior que trataba de validarse... solo me queda ver esto

**Comment by lopezjurip** [comment]
Claro, en la base de datos del recurso polimórfico, este debe tener una columna con el tipo y la id del recurso asociado. Aquí sale cómo se hace la migración:

http://guides.rubyonrails.org/association_basics.html#polymorphic-associations
